### ISSUE TITLE
Add config section to readme

### ISSUE THREAD

**Issue by SamVerschueren** [SamVerschueren/kap-s3#1]
We have to add a section to the readme that describes how to configure a plugin. This can only be done once we know how.

// @sindresorhus

**Comment by sindresorhus** [comment]
Yeah. We haven't decided where in the Plugins preference to put the "open config" button.

**Comment by SamVerschueren** [comment]
I see some things passing by regarding Kap 2. So I believe we can document this now.

**Comment by sindresorhus** [comment]
Blocked by https://github.com/wulkano/kap/issues/286

**Comment by mderazon** [comment]
what should go into the `path` field in the json config file ?

EDIT
nevermind, got it
`bucket-name/path`
